pkg/venafi/cloud/cloud.go

```diff
@@ -79,18 +75,23 @@ type certificateRequestClientInfo struct {
 }
 
 type certificateRequest struct {
-	CSR                          string                       `json:"certificateSigningRequest,omitempty"`
-	ZoneID                       string                       `json:"zoneId,omitempty"`
-	ExistingManagedCertificateId string                       `json:"existingManagedCertificateId,omitempty"`
-	ReuseCSR                     bool                         `json:"reuseCSR,omitempty"`
-	ApiClientInformation         certificateRequestClientInfo `json:"apiClientInformation,omitempty"`
-	ValidityPeriod               string                       `json:"validityPeriod,omitempty"`
+	CSR                      string                       `json:"certificateSigningRequest,omitempty"`
+	ApplicationId            string                       `json:"applicationId,omitempty"`
+	TemplateId               string                       `json:"certificateIssuingTemplateId,omitempty"`
+	CertificateOwnerUserId   string                       `json:"certificateOwnerUserId,omitempty"`
+	ExistingCertificateId    string                       `json:"existingCertificateId,omitempty"`
+	ApiClientInformation     certificateRequestClientInfo `json:"apiClientInformation,omitempty"`
+	CertificateUsageMetadata []certificateUsageMetadata   `json:"certificateUsageMetadata,omitempty"`
+	ReuseCSR                 bool                         `json:"reuseCSR,omitempty"`
+	ValidityPeriod           string                       `json:"validityPeriod,omitempty"`
 }
 
 type certificateStatus struct {
-	Id                        string                            `json:"Id,omitempty"`
-	ManagedCertificateId      string                            `json:"managedCertificateId,omitempty"`
-	ZoneId                    string                            `json:"zoneId,omitempty"`
+	Id string `json:"id,omitempty"`
+	//ManagedCertificateId      string                            `json:"managedCertificateId,omitempty"`
```

Comment:
I think we can remove this, since it will not be used.

Reply:
done